fhir-client/src/main/java/com/ibm/fhir/client/FHIRParameters.java

```diff
@@ -29,16 +31,18 @@
      * The valid set of modifiers that can be used when constructing the parameters for a search operation.
      */
     public enum Modifier {
```

Comment:
Mind adding a javadoc `@implNote please @see FHIRParameters.Modifier to keep in sync with SearchConstants.Modifier`  so we keep them in sync

Reply:
Updated.